core/org.testeditor.core/src/main/java/org/testeditor/core/services/impl/TestProjectServiceImpl.java

```diff
@@ -367,8 +398,8 @@ private File getDirectoryOfDemoProjects(String dir) throws IOException {
 						TestProjectConfig config = getProjectConfigFor(demoProjectName);
 						internalStoreProjectConfig(demoProjectName, config, true);
 					}
-					TestProject project = createProjectFrom(
-							new File(wsDir.getAbsolutePath() + File.separator + demoProjectDir.getName()));
+					TestProject project = createProjectFrom(new File(wsDir.getAbsolutePath() + File.separator
```

Comment:
I'm not a big fan of constructing instances of `File` like this. I know it's used all over the place, but why don't we use `new File(wsDir, demoProjectDir.getName())` here instead?